assets/app/features/upgrade/controllers/upgrade-backup.controller.spec.js

```diff
@@ -21,15 +21,22 @@ describe('Upgrade Flow - Backup Controller', function() {
         mockedDownloadResponse = {
             data: mockedDownloadFile,
             'headers': function() {}
+        },
+        upgradeVm = {
+            steps: {
+                activeStep: {
+                    finished: false
+                }
+            }
         };
 
     beforeEach(function() {
         //Setup the module and dependencies to be used.
         bard.appModule('crowbarApp');
         bard.inject('$controller', '$rootScope', '$q', '$httpBackend', '$document', 'crowbarBackupFactory');
-
+        $rootScope.upgradeVm = upgradeVm;
```

Comment:
Well, $rootScope is being defined as the ancestor of all scopes in an angular application, but given you're injecting it in a beforeEach section (that will be executed before each test is run), I'd expect to have a clean one each time.

I couldn't find a strong argument to use a new scope instead of the $rootScope for this case.